src/Sarif/Core/SarifLog.cs

```diff
@@ -101,8 +101,20 @@ public static SarifLog Load(Stream source, bool deferred = false)
                 return false;
             }
 
-            await Post(postUri, new MemoryStream(fileBytes), httpClient);
-            Console.WriteLine($"Posted log file successfully to: {postUri}");
+            HttpResponseMessage response = await Post(postUri, new MemoryStream(fileBytes), httpClient);
+            string responseText = response.Content.ReadAsStringAsync().Result;
```

Comment:
This method is actually async, so you can await this Task instead of doing .Result.  (And that is better.)